chapters/es/chapter1/1.mdx

```diff
@@ -0,0 +1,51 @@
+# Introducción
+
+## ¡Te damos la bienvenida al curso de 🤗!
+
+<Youtube id="00GKzGyWFEs" />
+
+Este curso te enseñará sobre procesamiento de lenguaje natural (PLN) usando librerías del ecosistema [Hugging Face](https://huggingface.co/) - [🤗 Transformers](https://github.com/huggingface/transformers), [🤗 Datasets](https://github.com/huggingface/datasets), [🤗 Tokenizers](https://github.com/huggingface/tokenizers) y [🤗 Accelerate](https://github.com/huggingface/accelerate) — así como el [Hub de Hugging Face](https://huggingface.co/models). El curso es completamente gratuito y sin anuncios.
```

Comment:
Would it make sense to use "bibliotecas" instead of "librerías" here (and elsewhere in the chapter)?

Reply:
Although the literal translation of library is "biblioteca", I have seen "librería" more often to describe libraries because of it's similarity to the word in english. 

However, I feel that this decision might need more input from other spanish-speaking programmers. In any case, I'll be sure to update it to biblioteca if deemed necessary 😄.